.appdata.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2023 Jakub Melka -->
+<component type="desktop-application">
+  <id>io.github.JakubMelka.Pdf4qt</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LGPL-3.0-only</project_license>
+  <name>PDF4QT</name>
+  <summary>Open Source PDF Editor</summary>
+  <description>
+    <p>
+      Unleash the power of PDF4QT: Your go-to open source PDF editor powered by the Qt framework. With a robust C++ library, intuitive PDF viewing/editing applications, and a handy command-line tool, PDF4QT streamlines your PDF interactions. Try it today.
+    </p>
+  </description>
+  <categories>
+    <category>Office</category>
+    <category>Utility</category>
+  </categories>
+  <developer_name>Jakub Melka</developer_name>
+  <url type="homepage">https://jakubmelka.github.io/</url>
+  <url type="bugtracker">https://github.com/JakubMelka/PDF4QT/issues</url>
+  <url type="help">https://github.com/JakubMelka/PDF4QT/discussions</url>
+  <update_contact>Mgr.Jakub.Melka_AT_gmail.com</update_contact>
+  <content_rating type="oars-1.1"/>
+  <launchable type="desktop-id">io.github.JakubMelka.Pdf4qt.Pdf4QtViewerProfi.desktop</launchable>
```

Comment:
launcheable should match the flatpak id.

You need to figure out which application you want to be the default and make it soo.

Reply:
Fixed.